src/main/java/io/fusionauth/http/util/HTTPTools.java

```diff
@@ -105,6 +123,22 @@ public static boolean isValueCharacter(byte ch) {
     return isURICharacter(ch) || intVal == ' ' || intVal == '\t' || intVal == '\n' || intVal >= 0x80;
   }
 
+  /**
+   * Build a {@link ParseException} that can be thrown.
+   *
+   * @param b     the byte that caused the exception
+   * @param state the current parser state
+   * @return a throwable exception
+   */
+  public static ParseException makeParseException(byte b, Enum<? extends Enum<?>> state) {
```

Comment:
This is just to allow for a consistent `ParseException` thrown from various places.